tests/test_p3_templatetags.py

```diff
@@ -0,0 +1,51 @@
+from datetime import timedelta
+
+import pytest
+
+from django.urls import reverse
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.utils import timezone
+
+from conference.models import TALK_STATUS, TALK_LEVEL
+from p3.templatetags import sessions
+from tests.factories import UserFactory, TalkFactory, ConferenceTagFactory, TalkSpeakerFactory, SpeakerFactory
+from tests.common_tools import get_default_conference, redirects_to, template_used, make_user
+
+pytestmark = [pytest.mark.django_db]
+
+
+def test_spealers_templatetag():
```

Comment:
(Little typo here, `spealers`->`speakers`)

Reply:
Snag! I'll send a fix for it :)